src/Orleans.CodeGeneration/Orleans.CodeGeneration.csproj

```diff
@@ -1,12 +1,12 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <PackageId>Microsoft.Orleans.OrleansCodeGenerator</PackageId>
+    <PackageId>Microsoft.Orleans.OrleansCodeGeneration</PackageId>
```

Comment:
I think there's a chance we'll drop runtime codegen, or to drastically change its shape, so this is a candidate to become stale